### ISSUE TITLE
How to setup Typeface in HICSSObject

### ISSUE THREAD

**Issue by ArunVicky001** [highcharts/highcharts-android#94]
Hi 

I have couple of fonts in my font folder which is available in the res and I want to setup those fonts for HILegend and HITitle but I don't see any style property to assign the Typeface object in HICSSObject

I tried like below but its not working 

        ```
        HICSSObject itemStyle = new HICSSObject();
        itemStyle.setFontWeight("1");
        itemStyle.setFontSize("10px");
        itemStyle.setColor("black");
        itemStyle.setFontFamily("font/lucidasans.ttf");

        HILegend legend = new HILegend();
        legend.setAlign("right");
        legend.setItemDistance(5);
        legend.setPadding(0);
        legend.setItemMarginBottom(-11);
        legend.setItemStyle(itemStyle);
```
      
Could you please tell how to set the up the custom fonts which is available in the font folder to `HITitle` and  `HILegend` ? 

or 

Could you tell how to apply resource file style to `HILegend` and `HITitle` ? 

Thanks

**Comment by soommy12** [comment]
Hello @ArunVicky001 ! Everything about custom fonts is described in [readme](https://github.com/highcharts/highcharts-android#custom-fonts).

**Comment by ArunVicky001** [comment]
Thanks for your response!

I saw the description it was very clear but if the latest version 
`compile 'com.highsoft.highcharts:highcharts:7.0.3'`  the custom fonts method is not coming  instead it available the version which you updated in the GitHub. 

could you please check and update addFont method in the dependency as well

Thanks

**Comment by ArunVicky001** [comment]
Am sorry the features is available in 7.1.1 version
